testing/simapp/app.go

```diff
@@ -196,6 +196,11 @@ type SimApp struct {
 	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
 	ScopedICAKeeper      capabilitykeeper.ScopedKeeper
 	ScopedIBCMockKeeper  capabilitykeeper.ScopedKeeper
+	ScopedICAMockKeeper  capabilitykeeper.ScopedKeeper
+
+	// make IBC modules public for test purposes
+	// these modules are never directly routed to by the IBC Router
+	ICAAuthModule ibcmock.IBCModule
```

Comment:
IBC routes to the top level IBC module of a middleware stack. In this case it would be the ica IBCModule. The ica IBCModule contains this ica auth module as an unexported variable. Thus to access it during testing I made it a public field of the SimApp. This will need to be done for all base middleware applications which cannot standalone (like transfer)

Reply:
:handshake:  We should make sure to note this in the developer docs. I'll make a note